src/main/scala/com/softwaremill/realworld/common/BaseEndpoints.scala

```diff
@@ -24,6 +24,11 @@ class BaseEndpoints(authService: AuthService):
     .securityIn(auth.http[String]("Token", WWWAuthenticateChallenge("Token")))
     .zServerSecurityLogic[Any, UserSession](handleAuth)
 
+  val secureOptionEndpoint: ZPartialServerEndpoint[Any, Option[String], Option[UserSession], Unit, ErrorInfo, Unit, Any] = endpoint
```

Comment:
I think the name could be better - that's an endpoint with optional security? Maybe `optionallySecureEndpoint`?

Reply:
ok